package.json

```diff
@@ -17,10 +17,7 @@
     "xkp": "cli.js",
     "xkpasswd": "cli.js"
   },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/vot/xkpasswd-node.git"
-  },
+  "repository": "https://github.com/vot/xkpasswd-node.git",
```

Comment:
i think this came up in some linter (may be), hence I split it. But I am okay both ways.